test/Driver/imported_modules_bridging_header.swift

```diff
@@ -1,2 +1,7 @@
+// RUN: not %target-swiftc_driver -typecheck %s -import-objc-header %S/Inputs/imported_modules/bridging_header.h -I %S/Inputs/imported_modules 2>&1 | %FileCheck %s
```

Comment:
Hm, %target-swift-build doesn't seem to exist; is %target-swift-frontend better?